lib/public_key/src/public_key.erl

```diff
@@ -702,8 +816,9 @@ decrypt_private(CipherText,
 %%--------------------------------------------------------------------
 %% Description: Public key decryption using the public key.
 %%--------------------------------------------------------------------
--doc(#{equiv => decrypt_public/3}).
--doc(#{since => <<"OTP R14B">>}).
+-doc(#{equiv => decrypt_public/3,
```

Comment:
```suggestion
-doc(#{equiv => decrypt_public(CipherText, Key, []),
```

You can write it like this and then it becomes very obvious in what way the function are equivalent.